WordPress/Classes/ViewRelated/Voice/AudioRecorderVisualizerView.swift

```diff
@@ -0,0 +1,71 @@
+import SwiftUI
+import AVFoundation
+
+struct AudioRecorderVisualizerView: View {
+    let recorder: AVAudioRecorder
+    @StateObject private var viewModel = AudioRecorderVisualizerViewModel()
+
+    var body: some View {
+        AudiowaveView(samples: viewModel.samples)
+            .onAppear {
+                viewModel.recorder = recorder
+            }
+    }
+}
+
+private struct AudiowaveView: View {
+    let samples: [Float]
+
+    var body: some View {
+        let indices = Array<Int>(samples.indices)
+        HStack(spacing: 6) {
+            ForEach(indices, id: \.self) { index in
+                let height = max(10, 80 * normalizePowerLevel(samples[index]))
+                Capsule(style: .continuous)
+                    .fill(Color(uiColor: .brand))
+                    .frame(width: 10, height: CGFloat(height))
+                    .animation(.spring(duration: 0.1), value: height)
+            }
+        }
+    }
+
+    private func normalizePowerLevel(_ power: Float) -> Float {
+        // About the expected range for voice
+        let minPower: Float = -42
```

Comment:
I learned about `dBFS` (`Decibels relative to full scale `), couldn't understand how it's measured at first 😀

Reply:
Tbh, I'm not 100% sure this is done correctly.